lib/examples.js

```diff
@@ -129,14 +128,25 @@ function getSourceFiles(type, name) {
 
         case 'handlebars':
             return Promise.all([
-                readFile(path.join(exampleDir, 'template.hbs'), 'utf8'),
-                readFile(path.join(exampleDir, 'context.js'), 'utf8')
-            ]).then(function (sourceFiles) {
-                return {
-                    template: sourceFiles[0],
-                    context : sourceFiles[1]
-                };
-            });
+                  readFile(path.join(exampleDir, 'template.hbs'), 'utf8'),
+                  readFile(path.join(exampleDir, 'context.js'), 'utf8')
+              ]).then(function (sourceFiles) {
+                  return {
+                      template: sourceFiles[0],
+                      context : sourceFiles[1]
+                  };
+              });
```

Comment:
Can we stick to 4-space tabs throughout the site? It seems these lines are just indentation changes.